test/components/interactive/dragBoxLayerTests.ts

```diff
@@ -50,6 +50,15 @@ describe("Interactive Components", () => {
       svg.remove();
     });
 
+    it("clipPath enabled", () => {
+      /*
+       * clipPath has to be enabled so the resize handle doesn't show on hovering
+       * over the edge of the drag box
+       */
```

Comment:
Let's put this comment either in the pull request description or in the source code as a comment.  Putting it here is not visible to anyone really.